src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/util/DateTimeHelper.kt

```diff
@@ -24,6 +25,13 @@ class DateTimeHelper {
       return formatter.print(DateTime(DateTimeZone.UTC)).toString()
     }
 
+    fun dateTimeWithDaylightSaving(): LocalDateTime {
+      val format = DateTimeFormat.forPattern("dd/MM/yyyy HH:mm")
+      val dateTimeString = format.print(DateTime(DateTimeZone.forTimeZone(TimeZone.getTimeZone("Europe/London")))).toString()
+      val formatter: DateTimeFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm")
+      return LocalDateTime.parse(dateTimeString, formatter)
+    }
```

Comment:
Couldn't you just do this?:

      `return LocalDateTime.now(ZoneId.of("Europe/London"))`

Reply:
Oops! Anyway, corrected!